docs/REST.md

```diff
@@ -330,6 +331,28 @@ Upload new student information based on an excel binary string.
 }
 ```
 
+```
+## PATCH `/api/student/item`
+
+Edit an item's duedate that is checked out to a student
+
+### Parameters
+
+Query string:
+* `studentId`: Id of the student who has the item
+
+Body:
+* `itemAddress`: The address of the item
+* `date`: The new date the item is due
```

Comment:
This is absolutely not a NACK. There are multiple other references to dates in the REST.md which we don't specify what format the date needs to be in. It also doesn't matter since moment.js can parse all of those that you listed.